hazelcast/src/main/resources/hazelcast-client-config-5.3.xsd

```diff
@@ -1186,4 +1187,26 @@
             <xs:enumeration value="RETRY_ALL"/>
         </xs:restriction>
     </xs:simpleType>
+
+    <xs:complexType name="alto">
+        <xs:annotation>
+            <xs:documentation>
+                Configures the Alto-aware client, which is the next generation Hazelcast
+                that uses thread-per-core model.
+                Alto-aware clients will maintain connections to all cluster members
+                along with connections to all Alto cores of each cluster member to
+                route invocations to the correct cores of the correct members in
+                best-effort basis.
```

Comment:
OK, simplified the documentation. I believe we have to say that it is best effort basis, because it really is. The partition table might be stale, or we might not have the connection at the time. Regarding the beta annotation. There is no concept of beta annotation for XSD or JSON schema. So, we have to explicitly write it for these two. For programmatic API, we put just the annotation as you suggested